src/platforms/javascript/common/configuration/micro-frontend-support/metadata-injection.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Metadata Injection
+sidebar_order: 10
+description: "Inject metadata into bundles which can later be used to identify which bundles may be responsible for an error."
+---
+
+Requires version `2.5.0` or higher of `@sentry/webpack-plugin`.
+
+Before you can use the <PlatformLink
```

Comment:
It sounds like this is simply a prereq step before users are able to use the ModuleMetadata integration, which actually does something useful for them.

Maybe this shouldn't be a separate page, and should simply be prereq steps on the ModuleMetadata Integration page.